optaplanner-examples/pom.xml

```diff
@@ -83,10 +83,6 @@
       <groupId>org.optaplanner</groupId>
       <artifactId>optaplanner-persistence-xstream</artifactId>
     </dependency>
-    <dependency><!-- Few examples use the JAXB integration -->
```

Comment:
Let's change this into a TODO, that we need an example that uses it :) That way we have an integration test too.